mail_notification_custom_subject/tests/test_mail_notification_custom_subject.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2020 Tecnativa - João Marques
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests import common
+
+
+class TestMailNotificationCustomSubject(common.TransactionCase):
+    def setUp(self):
+        super(TestMailNotificationCustomSubject, self).setUp()
+        self.partner_1 = self.env["res.partner"].create(
+            {
+                "name": "Test partner 1",
+                "supplier": True,
+            }
+        )
+        self.partner_2 = self.env["res.partner"].create(
+            {
+                "name": "Test partner 2",
+                "supplier": True,
+            }
+        )
+        # Delete current DB records for testing
+        self.env["mail.message.custom.subject"].search([]).unlink()
```

Comment:
```suggestion
```

Reply:
I was using this to get correct test results in case there are some records presents in the database beforehand that would produce different results. 
This transaction is then rolled back at the end of the test, as it is a `TransactionCase`, so it wouldn't affect the real DB. 
What is the correct way to do it, then?

Reply:
This is risky indeed. Perform a search before doing `message_post` and after, and then the difference will be the new created message.

Reply:
Thanks. I ended up deleting these lines, as suggested by @victoralmau as I misunderstood the problem :sweat_smile: